agent/engine/docker_image_manager_integ_test.go

```diff
@@ -313,7 +315,7 @@ func TestImageWithSameNameAndDifferentID(t *testing.T) {
 	task2.Containers[0].Image = identicalImageName
 	task3.Containers[0].Image = identicalImageName
 
-	err = renameImage(test3Image1Name, "testimagewithsamenameanddifferentid", "latest", goDockerClient)
+	err = renameImage(context.Background(), test3Image1Name, task1.Containers[0].Image, sdkDockerClient)
```

Comment:
Can you change this to the following to avoid dangling context? Same below.
```
ctx, cancel := context.WithCancel(context.TODO())
defer cancel()
err = renameImage(ctx, test3Image1Name, task1.Containers[0].Image, sdkDockerClient)
```